build/sections/partners.html

```diff
@@ -0,0 +1,43 @@
+<div class="bg-[url(../Images/BG_partners.webp)] bg-cover">
+    <div class="container pt-52">
+        <div class=" md:w-[50rem] pb-16">
+            <p class="font-raleway font-bold text-3xl pb-4 md:text-4xl md:pb-5">
+                MaibornWolff x The Gym x Maliyo 
+            </p>
+            <p class="font-montserrat font-medium text-base md:text-xl">
+                Für das Training werden die Erfahrung und das Wissen über erfolgreiche Softwareprojekte kombiniert mit dem standortspezifischen ...
+            </p>
+        </div>  
+        <div class=" flex flex-col px-[25%] space-y-6 md:grid grid-cols-2  md:pr-28 md:gap-8 lg:gap-8 lg:pl-40 lg:pr-28 py-9">
+            <div>
+                <img class="w-64 h-36" src="Images/mailb.svg" alt="MaibornWolff">
+            </div>  
+            <div class="md:-translate-y-14">
+                <div class=" space-y-12 px-6 pb-2 md:flex md:gap-5 md:space-y-0 md:px-0">
```

Comment:
As i mentioned above remove the unnecessary spaces
```suggestion
                <div class="space-y-12 px-6 pb-2 md:flex md:gap-5 md:space-y-0 md:px-0">
```